bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/ShellyBindingConstants.java

```diff
@@ -0,0 +1,321 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.shelly.internal;
+
+import java.util.Collections;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+
+/**
+ * The {@link ShellyBindingConstants} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author Markus Michels - Initial contribution
+ */
+@NonNullByDefault
+public class ShellyBindingConstants {
+
+    public static final String            VENDOR                           = "Shelly";
```

Comment:
Why do you use such a weired format in this and all other files?

Reply:
?

Reply:
Not direct. This means that I need to tag all members as @Nullable or initialize. Let's keep this on the list after the release. I don't want to make cosmetical changes before RC1. Some of the members might become null, because the device doesn't report those (depending on the device type) and that could lead into a wrong null check.